bin/package_build.py

```diff
@@ -249,6 +250,24 @@ def rockylinux():
 		DATA.close()
 		print(f"Saved!\nfilename: {q}")
 
+def zOSOpenTools():
+	url = "https://api.github.com/users/ZOSOpenTools/repos"
```

Comment:
If you use this URL, you'll not find all repository names in your response as you can see in your `ZOSOpentools.json`. Refer to [this](https://stackoverflow.com/questions/18647031/list-all-github-repos-for-an-organization-including-those-in-teams).